### DISCUSSION TITLE
Weather Radar

### DISCUSSION THREAD

**Discussion by itsj4y** [home-assistant/frontend#10853] [Dashboard]
Is there a way to see a live weather radar on the map card and still retain the entity tracking? Something like this https://github.com/rwev/leaflet-radar.

**Comment by Misiu** [comment]
yes, this is possible. Take a look at: https://www.home-assistant.io/lovelace/iframe/
Basically, you can embed any webpage, so find one that suits your needs, and you're ready to go.

**Reply by itsj4y** [reply]
So how would I retain the entity tracking like the map card on an embedded webpage? Basically what I want is just an overlay added onto the map card that shows the most up to date radar image
